src/types/analytics.ts

```diff
@@ -1,4 +1,4 @@
-export type AnalyticStats = {
+export type AnalyticStatsDTO = {
```

Comment:
nit: rename this to `AnalyticStatsDto` to better follow camelCase conventions where only the first character of an abbreviation is capitalized.

Reply:
This is opinionated, but the AirB&B guide begs to differ:
https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms

Reply:
do we have an established convention in the codebase already?

Reply:
Yea, as discussed we use `findSomethingById` rather than `findSomethingByID` so hehehe